modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -49,6 +49,12 @@ public class UpgradeCatchAllCheck {
 		assetCategory.getLeftCategoryId;
 	}
 
+	public void method(CommerceAddress commerceAddress) {
+		_commerceAddressLocalService.updateCommerceAddress(commerceAddress);
+		_commerceAddressLocalService.updateCommerceAddress(null);
```

Comment:
@MicaelleSilvaa, do we really need this line? If not, then we can remove it. Otherwise, ideally, we should use the parameter `null` in the `_commerceAddressService` too, like:
```
_commerceAddressService.updateCommerceAddress(commerceAddress);
_commerceAddressService.updateCommerceAddress(null);
```
This is important to keep consistency. I mean, if I'm testing the `*LocalService` with `commerceAddress` and `null` then I should do the same for the `*Service`.

Reply:
In this case, you don't need to test more than one class.